src/mito2/src/worker.rs

```diff
@@ -373,6 +373,12 @@ async fn write_cache_from_config(
     // TODO(yingwen): Remove this and document the config once the write cache is ready.
     warn!("Write cache is an experimental feature");
 
+    tokio::fs::create_dir_all(Path::new(&config.experimental_write_cache_path))
+        .await
+        .context(CreateDirSnafu {
+            dir: &config.experimental_write_cache_path,
+        })?;
```

Comment:
nitpick: Only do this when `config.enable_experimental_write_cache` is true

Reply:
Yes, it's checked above.